docs/src/main/asciidoc/spring-cloud-commons.adoc

```diff
@@ -362,8 +362,7 @@ for details of how the `RestTemplate` is set up.
 ==== Retrying Failed Requests
 
 A load balanced `RestTemplate` can be configured to retry failed requests.
-By default this logic is disabled, you can enable it by setting
-`spring.cloud.loadbalancer.retry.enabled=true` and adding link:https://github.com/spring-projects/spring-retry[Spring Retry] to your application's classpath.  The load balanced `RestTemplate` will
+By default this logic is disabled, you can enable it by adding link:https://github.com/spring-projects/spring-retry[Spring Retry] to your application's classpath.  The load balanced `RestTemplate` will
```

Comment:
Add "To disable set `spring.cloud.loadbalancer.retry.enabled=false`." or something similar.